pkg/operator/starter.go

```diff
@@ -165,6 +165,22 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 			return isVSphere
 		},
 		nil,
+	).WithConditionalResources(
+		bindata.Asset,
+		[]string{
+			"assets/kube-controller-manager/gce/cloud-provider-role.yaml",
+			"assets/kube-controller-manager/gce/cloud-provider-binding.yaml",
+			"assets/kube-controller-manager/sa.yaml",
```

Comment:
this sa should be kept. I think the sa for cloud-provider comes from kube

Reply:
I missed this one. Thanks for noticing it.

Reply:
Thanks for the review @atiratree  :) 

> I think the sa for cloud-provider comes from kube

Does this mean there's a SA somewhere else I need to remove?

Reply:
I think the SA should be created automatically when the controller runs

https://github.com/kubernetes/kubernetes/blob/a3adc759a3c096accafb1694f77362ee5f957a22/staging/src/k8s.io/legacy-cloud-providers/gce/gce.go#L652

so, no action required

Reply:
thanks! :)